packages/containers/src/HeaderBar/HeaderBar.sagas.test.js

```diff
@@ -70,17 +70,23 @@ describe('HeaderBar sagas', () => {
 	});
 
 	describe('handleOpenProduct', () => {
-		global.open = jest.fn();
+		const result = { opened: {} };
+		global.open = jest.fn(() => result.opened);
```

Comment:
You don't need to init those values here, it's done in beforeEach. Just declare `result`